ios/FluentUI/Extensions/UIColor+Extensions.swift

```diff
@@ -58,3 +58,14 @@ import UIKit
         }
     }
 }
+
+// MARK: - Other internal convenience extensions
+
+extension UIColor {
```

Comment:
we need this for what? also, potential clash when integrated to dev main?

Reply:
Ah good catch, we don't need this anymore in later changes (older versions of this branch had hardcoded colors).

Will remove, but even if it stayed I don't think it would have clashed with devmain because it's all `internal` code, nothing public.